src/controller/cluster.js

```
@@ -0,0 +1,17 @@
+const bcrypt = require('bcryptjs');
+const envs = require("../config/env/index");
+const handleError = require("../middleware/handleError");
+
+const cluster = async (req, res, next) => {
+
+  try{
+
+  }
+  catch(err){
+    next(err);
+  }
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)